index.js

```diff
@@ -28,9 +28,10 @@ var adminDVPermission = require('permission')(['admin', 'data-viewer']);
 
 var app = express();
 
+var disableAuth = process.env.DISABLE_AUTH == 1 || process.env.DISABLE_AUTH == 'true';
```

Comment:
Prefer `===`.

Reply:
So it should be `process.env.DISABLE_AUTH === '1' || process.env.DISABLE_AUTH === 'true';`, right? On my tests the ENV variables are always received as strings. I thought that it may be different in another systems.